osu.Framework/Graphics/Containers/ScrollContainer.cs

```diff
@@ -128,7 +128,7 @@ public bool ScrollbarOverlapsContent
         /// <summary>
         /// The maximum distance that the scrollbar can move in the scroll direction.
         /// </summary>
-        public float ScrollbarMovementExtent => Math.Max(DrawSize[ScrollDim] - Scrollbar.DrawSize[ScrollDim], 0);
+        public float ScrollbarMovementExtent => Math.Max(ChildSize[ScrollDim] - Scrollbar.DrawSize[ScrollDim], 0);
```

Comment:
`DisplayableContent` also translates to `ChildSize[ScrollDim]`, and would make this easy to parse.
```suggestion
        public float ScrollbarMovementExtent => Math.Max(DisplayableContent - Scrollbar.DrawSize[ScrollDim], 0);
```